elm/js/javascript.js

```diff
@@ -0,0 +1,28 @@
+var node = document.getElementById('app');
+var storedName = localStorage.getItem('name');
+var socket = io();
```

Comment:
could _some_ of this have been included from the existing file (_i.e. avoid duplication_) ?